pkg/utils/utils.go

```diff
@@ -27,3 +29,14 @@ func JoinInt(nums []int, delim string) string {
 	}
 	return strings.TrimSuffix(result, delim)
 }
+
+// IsFlagSetByUser check if the a flag is set by user explicitly
+func IsFlagSetByUser(flagSet *pflag.FlagSet, flagName string) bool {
+	setByUser := false
```

Comment:
It's not necessary to use a named variable here, just return true and false would be enough.